### ISSUE TITLE
Import bookmark tags from the browser

### ISSUE THREAD

**Issue by mattijohn** [calpaterson/quarchive#63]
# Brief overview

AS A user

I WANT bookmark tags to be imported from the browser

SO THAT I don't need to create tags in the Quarchive website instead of in the browser

# Additional details

**Comment by calpaterson** [comment]
Firefox doesn't yet let extensions access this data (they track this as [bug 1225916](https://bugzilla.mozilla.org/show_bug.cgi?id=1225916)) but perhaps will in the future.  

I'm not aware that chrome allows for tagging bookmarks via the browser UI.

I half-plan to add an [address bar button](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Page_actions) to the browser extension to allow people to at least edit the tags there.  The other (and main) feature of that would be making it more obvious that you're visiting a page that you've bookmarked.  Maybe it could show you "$n comments" as well.

**Comment by mattijohn** [comment]
An address bar button would be great. TBH I was only trying to add tags in Firefox to work around the bug with editing tags. Maybe should be changed to say something like 

AS A user

I WANT be able to add bookmark tags within the browser

SO THAT I don't need to create tags in the Quarchive website

**Comment by calpaterson** [comment]
Agreed and done - will be a bit of a while until I broach the subject of an address bar button though I'm afraid!

**Comment by iaeiou** [comment]
Hi @calpaterson 

> Agreed and done

Could you please confirm that tags are properly imported now?

It would be a super feature, tags and Firefox are such a mess!

**Comment by calpaterson** [comment]
I'm sorry, when I said "agreed and done" I wasn't referring to the work to complete this story.

This will be one of the next things I work on though.
